src/main/java/com/sparta/gourmate/domain/user/controller/AddressController.java

```diff
@@ -0,0 +1,43 @@
+package com.sparta.gourmate.domain.user.controller;
+
+import com.sparta.gourmate.domain.user.dto.AddressRequestDto;
+import com.sparta.gourmate.domain.user.dto.AddressResponseDto;
+import com.sparta.gourmate.domain.user.service.AddressService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/addresses")
+public class AddressController {
+
+    private final AddressService addressService;
+
+    // 주소 등록
+    @PostMapping("/{userId}")
+    public AddressResponseDto createAddress(@PathVariable Long userId, @RequestBody @Valid AddressRequestDto requestDto) {
```

Comment:
```suggestion
    public AddressResponseDto createAddress(@AuthenticationPrincipal UserDetailsImpl userDetails, @RequestBody @Valid AddressRequestDto requestDto) {
```

서영님 고생많으셨어요 ☺️
회의 때 말씀 드린 것처럼 인증객체 받아오셔서 사용하시면 돼요!